src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -76,6 +77,11 @@ public Window DynamoWindow
             }
         }
 
+        /// <summary>
+        /// A reference the PreferenceSettings as set by the Dynamo > Preferences window
+        /// </summary>
+        public PreferenceSettings PreferenceSettings { get; }
```

Comment:
this is already available on startup params, which are available here - `loadedParams.StartupParams. Preferences`
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Extensions/StartupParams.cs#L25

which returns an interface - it can be cast to a PreferenceSettings concrete type to get the new property you've added.

`IPreferenceSettings` cannot be changed at this time unfortunately for backward compatibility reasons.

Theres a few options here like:
* obsoleting Preferences and replacing it with PreferenceSettings
* creating a new Interface like IPreferenceSettingsRequiredProps and asking users to cast to that

Theres probably others - but we shouldn't just add a new property pointing to the same object.